robots/FrankaEmikaPandaRobot.m

```diff
@@ -0,0 +1,61 @@
+% (C) Copyright 2011-2019 DQ Robotics Developers
+% 
+% This file is part of DQ Robotics.
+% 
+%     DQ Robotics is free software: you can redistribute it and/or modify
+%     it under the terms of the GNU Lesser General Public License as published by
+%     the Free Software Foundation, either version 3 of the License, or
+%     (at your option) any later version.
+% 
+%     DQ Robotics is distributed in the hope that it will be useful,
+%     but WITHOUT ANY WARRANTY; without even the implied warranty of
+%     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%     GNU Lesser General Public License for more details.
+% 
+%     You should have received a copy of the GNU Lesser General Public License
+%     along with DQ Robotics.  If not, see <http://www.gnu.org/licenses/>.
+%
+% DQ Robotics website: dqrobotics.sourceforge.net
+%
+% Contributors to this file:
+%     Juan Jose Quiroz Omana - juanjqogm@gmail.com
+
+classdef FrankaEmikaPandaRobot
+    methods (Static)
+        function ret = kinematics()
+            %Modified D-H of FrankaEmikaPanda
```

Comment:
What's the justification for using the modified DH convention?

Reply:
I used it since the [official documentation](https://frankaemika.github.io/docs/control_parameters.html) provided by the manufacturer Franka Emika uses the modified DH parameters. On the other hand, I modeled the robot using the standard convention to check the class DQ_SerialManipulatorMDH. So, we have both models available.